samples/esb/prx/src/main.c

```diff
@@ -109,6 +109,13 @@ int clocks_start(void)
 		return err;
 	}
 
+	/* Block until clock is started.
+	 */
+	while (clock_control_get_status(clk, CLOCK_CONTROL_NRF_SUBSYS_HF) ==
+		CLOCK_CONTROL_STATUS_STARTING) {
```

Comment:
Should be `CLOCK_CONTROL_STATUS_ON` instead, if we want to be sure that HFCLK has started.

Reply:
Fixed. Both PTX and PRX are confirmed to work with the change.